back/test/integration_tests/test_cli.py

```diff
@@ -18,17 +19,20 @@
 # time to wait for Auth0 database to index updated data fields
 WAIT = 5
 
+test_id_static_suffix = "-TEST"
+test_id_suffix = f"{date.today().isoformat()}-{random.randint(0, 1000000)}{test_id_static_suffix}"
 
 @pytest.fixture
 def auth0_roles(auth0_management_api_client):
     # Set up test roles in Auth0
     interface = auth0_management_api_client._interface
+    #generate test id suffix with datetime and a uuid guaranteed to be increasing
```

Comment:
can you move this comment to line 23 above? Also not sure how correct it still is since no UUID is used but instead a random int.
Can you maybe add that we do this to guarantee unique role names in Auth0?

Reply:
Sorry, that was a bad rejig. I'll fix up with a clearer comment.